tests/snapshots/TestCLICommands_atmos_describe_config_imports.stdout.golden

```diff
@@ -0,0 +1,166 @@
+                        - varfiles/$PROJECT_NAME.tfvars.json
+                apply_requirements:
+                    - approved
+        workflow_templates:
+            workflow-1:
+                apply:
+                    steps:
+                        - run: terraform apply $PLANFILE
+                plan:
+                    steps:
+                        - run: terraform init -input=false
+                        - run: terraform workspace select $WORKSPACE || terraform workspace new $WORKSPACE
+                        - run: terraform plan -input=false -refresh -out $PLANFILE -var-file varfiles/$PROJECT_NAME.tfvars.json
+    github:
+        gitops:
+            infracost-enabled: false
+            opentofu-version: 1.8.4
+            terraform-version: 1.9.8
+schemas:
+    jsonschema:
+        base_path: stacks/schemas/jsonschema
+    opa:
+        base_path: stacks/schemas/opa
+templates:
+    settings:
+        enabled: true
+        sprig:
+            enabled: true
+        gomplate:
+            enabled: true
+            timeout: 5
+            datasources: {}
+        evaluations: 1
+settings:
+    list_merge_strategy: replace
+    terminal:
+        max_width: 120
+        pager: true
+        colors: true
+        unicode: true
+        syntax_highlighting:
+            enabled: true
+            lexer: ""
+            formatter: terminal
+            theme: dracula
+            pager: false
+            line_numbers: true
+            wrap: false
+    markdown:
+        document:
+            color: ${colors.text}
+        heading:
+            color: ${colors.primary}
+            bold: true
+        strong:
+            color: ${colors.secondary}
+            bold: true
+        emph:
+            color: ${colors.muted}
+            italic: true
+        code_block:
+            color: ${colors.secondary}
+            margin: 1
+        link:
+            color: ${colors.primary}
+            underline: true
+    inject_github_token: true
+vendor:
+    base_path: ./vendor.yaml
+initialized: true
+stacksBaseAbsolutePath: /absolute/path/to/repo/tests/fixtures/scenarios/atmos-cli-imports/stacks
+includeStackAbsolutePaths:
+    - /absolute/path/to/repo/tests/fixtures/scenarios/atmos-cli-imports/stacks/deploy/**/*
+excludeStackAbsolutePaths:
+    - /absolute/path/to/repo/tests/fixtures/scenarios/atmos-cli-imports/stacks/**/_defaults.yaml
+terraformDirAbsolutePath: /absolute/path/to/repo/tests/fixtures/scenarios/atmos-cli-imports/components/terraform
+helmfileDirAbsolutePath: /absolute/path/to/repo/tests/fixtures/scenarios/atmos-cli-imports/components/helmfile
+default: false
+version:
+    check:
+        enabled: true
+        timeout: 1000
+        frequency: 1h
+validate:
+    editorconfig:
+        format: default
+        color: true
+cli_config_path: /absolute/path/to/repo/tests/fixtures/scenarios/atmos-cli-imports
+import:
+    - https:/raw.githubusercontent.com/cloudposse/atmos/refs/heads/main/atmos.yaml
```

Comment:
Is this typo deliberate? If not deliberate then we have a problem with tests passing and not catching the invalid URL

Reply:
This is a result of how we generate and sanitize the snapshots across platforms, and not an actual bug. It will get fixed in another PR.